courses/free-courses-vi.md

```diff
@@ -150,6 +151,11 @@
 * [jQuery Cơ bản](https://www.youtube.com/playlist?list=PLRhlTlpDUWsyAGY7FDGSndEhOD3F2Ruhm) - TEDU
 
 
+### Machine-Learning
+
+* [Machine learing cơ bản](https://machinelearningcoban.com) - Vũ Hữu Tiệp
```

Comment:
:thinking: It seems a blog / website with a solid topic.

Reading [CONTRIBUTING guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#genres-we-dont-list)...
> Because the Internet is vast, we don't include in our lists:
> - blogs
> - blog posts
> - articles
> - websites (except for those that host LOTS of items that we list.)
>...

:information_source: I don't know if discard or move it to book listing

Reply:
This's a  blog provide course for Machine Learning topic written in Vietnamese.
Because I saw only this file is using for Vietnamese language and most suitable to place this.

Reply:
looks to me like it's an in-progress book. We have a notation for that! 🚧